observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,31 @@ These release notes track the development of the Network Observability Operator
+* Previously, when Loki was disabled, and the OpenShift web console was first loading, an error would occur: `Request failed with status code 400 Loki is disabled`. With this fix, the errors no longer occur. (link:https://issues.redhat.com/browse/NETOBSERV-1706[*NETOBSERV-1706*])
+* Previously, in the *Topology* view of the OpenShift web console, when clicking on the *Step into* icon next to any graph node, the filters were not applied as required in order to set the focus to the selected graph node, resulting in showing a wide view of the *Topology*. Now the filters are correctly set, effectively narrowing down the *Topology*. As part of this change, clicking the *Step into* icon on a *Node* now brings you to the *Resource* scope instead of the *Namespaces* scope. (link:https://issues.redhat.com/browse/NETOBSERV-1720[*NETOBSERV-1720*])
+* Previously, when Loki was disabled, in the *Topology* view of the OpenShift web console with the *Scope* set to *Owner*, clicking on the *Step into* icon next to any graph node would bring the *Scope* to *Resource*, which isn't available without Loki, so an error message was shown. Now the *Step into* icon is hidden in the *Owner* scope when Loki is disabled, so this scenario no longer occurs.(link:https://issues.redhat.com/browse/NETOBSERV-1721[*NETOBSERV-1721*])
+* Previously, when Loki was disabled, an error was displayed in the *Topology* view of the OpenShift web console when a group was set, but then the scope was changed so that the group becomes invalid. Now the invalid group is removed, preventing the error. (link:https://issues.redhat.com/browse/NETOBSERV-1722[*NETOBSERV-1722*])
+* When creating a `FlowCollector` resource from the OpenShift web console *Form view*, as opposed to the *YAML view*, the following settings are incorrectly managed by the web console: `agent.ebpf.metrics.enable` and `processor.subnetLabels.openShiftAutoDetect`. These settings can only be disabled in the *YAML view*, not in the *Form view*. To avoid any confusion, these settings have been removed from the *Form view*. They are still accessible in the *YAML view*. (link:https://issues.redhat.com/browse/NETOBSERV-1731[*NETOBSERV-1731*])
+* Previously, the eBPF agent was unable to clean up traffic control flows installed before an ungraceful crash, for example a crash due to a SIGTERM signal. This led to the creation of multiple traffic control flow filters with the same name, since the older ones were not removed. With this fix, all previously installed traffic control flows are cleaned up when the agent starts, before installing new ones. *NETOBSERV-1732*
```

Comment:
This entry is missing a link to the bug

Reply:
Its bc this is a private jira, so I had to remove it.

Reply:
Just confirmed that this Jira can be set to a lower security level, as its already been customer-facing issue.